daemon/daemon.cc

```diff
@@ -269,6 +270,7 @@ auto onFileAdded(tr_session* session, std::string_view dirname, std::string_view
 
 void printMessage(
     FILE* ostream,
+    std::chrono::time_point<std::chrono::system_clock> now,
```

Comment:
Here and everywhere else:
```suggestion
    std::chrono::system_clock::time_point now,
```